app/assets/stylesheets/application.scss

```diff
@@ -13,6 +13,7 @@
  *= require_self
  */
 
+@import "variables";
```

Comment:
I think materialize also has 'variables' that may interfere, we should import this after all other libraries so we can be sure the libraries wont override our custom stuff